src/main/resources/db/postgresql/V1_19__CREATE_WORKSTACK_COLUMN.sql

```diff
@@ -0,0 +1,26 @@
+DROP TABLE IF EXISTS workstack_column;
```

Comment:
should this file be included?

Reply:
yes, as discussed it's for flyway consistency when we merge wcs branch back to cs